website/guides/terraform-provider-development-program.html.md

```diff
@@ -0,0 +1,226 @@
+containing the code requesting the final code review. HashiCorp will review the
+code and provide feedback about any changes that may be required.  This is often
+an iterative process and can take some time to get done.
+
+The vendor is also required to provide access credentials for the infrastructure
+(cloud or other) that is managed by the provider. Please encrypt the credentials
+using our public GPG key published at keybase.io/terraform (you can use the form
+at https://keybase.io/encrypt#terraform) and paste the encrypted message into
+the [webform](https://goo.gl/forms/iqfz6H9UK91X9LQp2). Please do NOT enter
+plain-text credentials. These credentials are used during the review phase,
+as well as to test the provider as part of the regular testing HashiCorp conducts.
+
+->
+**NOTE:** It is strongly recommended to develop support for just one or two resources first and go through the review cycle before developing support for all the remaining resources. This approach helps catch any code construct issues early, and avoids the problem from multiplying across other resources.  In addition, one of the common gaps is often the lack of a complete set of acceptance tests, which results in wasted time. It is recommended that you make an extra pass through the provider code and ensure that each resource has an acceptance test associated with it.
+
+### 5. Release
+
+At this stage, it is expected that the provider is fully developed, all tests
+and documentation are in place,the acceptance tests are all passing, and that
+HashiCorp has reviewed the provider.
+
+HashiCorp will create a new GitHub repo under the terraform-providers GitHub
+organization for the new provider (example: `terraform-providers/terraform-provider-NAME`)
+and grant the owner of the original provider code write access to the new repo.
+A GitHub Pull Request should be created against this new repo with the provider
+code that had been reviewed in step-4 above. Once this is done HashiCorp will
+review and merge the PR, and get the new provider listed on
+[terraform.io](https://www.terraform.io). This is also when the provider
+acceptance tests are added to the HashiCorp test harness (TeamCity) and tested
+at regular intervals.
+
+Vendors whose providers are listed on terraform.io are permitted to use the
+[HashiCorp Tested logo](/assets/images/docs/hashicorp-tested-icon.png) for their provider.
+
+### 6. Support
+
+Many vendors view the release step to be the end of the journey, while at
+HashiCorp we view it to be the start. Getting the provider built is just the
+first step in enabling users to use it against the infrastructure. Once this is
+done on-going effort is required to maintain the provider and address any
+issues in a timely manner.
+
+The expectation is to resolve all critical issues within 48 hours and all other
+issues within 5 business days. HashiCorp Terraform has as extremely wide
```

Comment:
I like that we set expectations but 48 hours and 5 business days seems really hard to place on Vendors. Not a blocker though.

Reply:
Agreed, its easier to loosen this over time but for our initial set of official providers we are requesting this. 👍

Reply:
👍